libs/core/src/lib/form/form-message/form-message.component.ts

```diff
@@ -24,4 +24,11 @@ export class FormMessageComponent {
     /** Whether message should be in static mode, without popover. It's mostly used for forms component, that contain dropdown */
     @Input()
     static = false;
+
+    /**
+     * Whether message is used inside popovers or dialogs.
+     * When it is enabled box shadow is removed and message is expanded to whole container width
+     */
+    @Input()
```

Comment:
`form-message` is by default displayed as inline element. Maybe we should name it `embedded` or style based on
`.fd-popover__body > .fd-form-message`

Reply:
Changing name to `embedded` will be good